src/app/contextBroker/contextBroker.cpp

```diff
@@ -252,6 +254,7 @@ bool            https;
 bool            mtenant;
 char            rush[256];
 char            allowedOrigin[64];
+char            corsEnabled[64];
```

Comment:
Just to check if I'm understading correctly how this would work...

* If `-corsOrigin foo` is used, then Orion adds `Access-Control-Allow-Origin: foo` to the response of all GET operations.
* If `-enableCORS bar` is used, then Orion adds `Access-Control-Allow-Origin: bar` to the responses of all OPTIONS operations (*) along with others headers: `Access-Control-Allow-Headers` (with hardwired value), `Access-Control-Max-Age` (hardwired value) and `Access-Control-Allow-Methods` (value dedpending on the methods supported by the URL).

Is my understanding correct?

(*) Well, not in this PR, that covers only two URLs, but at the end it will be that way :)

Reply:
Well yes and no :) 

Considering what ´-corsOrigin foo´ does, ´-enableCORS foo´ (going to be -cors upon @kzangeli's suggestion) does the exact same thing. But it also adds the rest of the headers that should be there in a proper response to a preflight request.

I decided not to change ´-corsOrigin´ to ´-cors´ directly in case there are people using it and we might want to go with a soft roll-out, removing the -corsOrigin switch in a latter PR.

Since we are going to remove the -corsOrigin switch once we have all the required changes for #501 anyway, I can do it in a commit directly if this is more appropriate. 

I am open to suggestions :)

Reply:
My point is that, with the current implementation you can have different `Access-Control-Allow-Origin` for GET responses and for OPTIONS responses. Does that make sense in CORS? If not, then we should unify to just one switch.

Reply:
...and I'd suggest to that the unification in this PR (as I understand it is easy to do).

Reply:
Yes, you are right, there is no defense mechanism against setting two different origins. I will remove the old switch in this PR.

Reply:
Instead of removing the old switch... what about of reusing it? That way current Orion users with `contextBroker ... -corsOrigin foo` will not break when upgrading Orion version.

Reply:
I thought about reusing it but the naming might be a problem since we are not only setting the CORS allowed origin but also the max-age.

Having said that, is it currently possible to have more than one parameter for a switch? If yes, I would prefer ´-cors __ALL 86400´ for example. If not, should we go for something like ´-corsOrigin __ALL -corsMaxAge 86400´?

Reply:
No, I think it is not possible. The "trick" we have used in similar cases is using some character as separator, e.g. ":"

```
-notificationMode threadpool:600:20
```

However, in this case I think is clearer having two switches, as you cite:

```
-corsOrigin __ALL -corsMaxAge 86400
```

With defaults would be:

* `-corsOrigin`: CORS not enabled at all (i.e. no domain header in GET responses, no routes for OPTIONS verb)
* `-corsMaxAge` 86400 (24 hours)

Reply:
Fixed in 8c48f60